kdc/kdc.8

```diff
@@ -155,6 +155,20 @@ Permit tickets with no addresses.
 This option is only relevant when check-ticket-addresses is TRUE.
 .It Li allow-anonymous = Va boolean
 Permit anonymous tickets with no addresses.
+.It Li historical_anon_realm = Va boolean
+Enables pre-7.0 non-RFC-comformant KDC behavior.
+With this option set to
+.Li true
+the client realm in anonymous pkinit as-replies will be the requested realm,
+rather than the RFC-conformant
+.Li WELLKNOWN:ANONYMOUS
+realm.
+This can have a security impact on servers that expect to grant access to
+anonymous, but authenticated to the KDC users of the realm in question.
+They would also grant access to unauthenticated anonymous users.
+The
+.Li true
+settings is not recommended.
 .It Li max-kdc-datagram-reply-length = Va number
```

Comment:
Perhaps reword something like the following?

With this option set to true, the client realm in anonymous PKINIT AS replies will be the requested realm, rather than the well known anonymous name. Unauthenticated anonymous principals will appear to services as authenticated anonymous. As such, it is not recommend to set this option to true.

Reply:
Done in the next update.